### ISSUE TITLE
CMake find module

### ISSUE THREAD

**Issue by ghisvail** [NFFT/nfft#12]
I wrote a CMake find module to help the discovery of the different components of NFFT v3.3 inside projects using CMake as their build system. Would you guys be interested to have it upstream in the repository ?

**Comment by jenskeiner** [comment]
Sorry, didn't take note of this issue until now. Will consider that and let you know. Thanks.

**Comment by ghisvail** [comment]
Best might be to submit it upstream to Kitware

**Comment by jenskeiner** [comment]
Moin Moin,

ich habe die Travis-Konfiguration aktualisiert, so dass die Tests in einem Docker-Container laufen können. Im Zweig „travis-ci" funktioniert es damit wieder.

Viele Grüße,


Jens

**Comment by mpip** [comment]
Ok, danke. Mal schauen ob ich das auch für meine anderen Repos verwenden 
kann.

Viele Grüße und frohe Weihnachten,
Michael


Am 22.12.2015 um 20:21 schrieb Jens Keiner:
> Moin Moin,
>
> ich habe die Travis-Konfiguration aktualisiert, so dass die Tests in 
> einem Docker-Container laufen können. Im Zweig „travis-ci" 
> funktioniert es damit wieder.
>
> Viele Grüße,
>
>
> Jens
> > Am 21.12.2015 um 22:29 schrieb Ghislain Antony Vaillant 
> <notifications@github.com>:
> >
> > Best might be to submit it upstream to Kitware
> >
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NFFT/nfft/issues/12#issuecomment-166708054>.
>


-- 
**********************************************************
Michael Pippig
eMail: 	michael.pippig.-tuc@gmail.com
WWW: 	http://www.tu-chemnitz.de/~potts/workgroup/pippig
**********************************************************

**Comment by jenskeiner** [comment]
Moin Moin,

in Kürze würde ich das NFFT-Repository nochmals neu aufsetzen, z.B. auch, damit die SVN-Namen korrekt auf GitHub-Benutzer abgebildet werden. Es wäre vorteilhaft, vorher alle Zweige wieder in den Hauptzweig (als nfft-3.3) zu integrieren. Können wir das hier jetzt machen? Ich nehme an, dass der Zweig nur in der Git-Welt existiert...

Viele Grüße und ebenfalls frohe Festtage,


Jens

**Comment by mpip** [comment]
Ja, der Zweig existiert nur im Git-Repo um die Travis-CI Integration zu 
testen. Kann jetzt in den master wandern und der travis-ci Branch kann 
gelöscht werden.

Viele Grüße,
Michael

Am 23.12.2015 um 13:51 schrieb Jens Keiner:
> Moin Moin,
>
> in Kürze würde ich das NFFT-Repository nochmals neu aufsetzen, z.B. 
> auch, damit die SVN-Namen korrekt auf GitHub-Benutzer abgebildet 
> werden. Es wäre vorteilhaft, vorher alle Zweige wieder in den 
> Hauptzweig (als nfft-3.3) zu integrieren. Können wir das hier jetzt 
> machen? Ich nehme an, dass der Zweig nur in der Git-Welt existiert...
>
> Viele Grüße und ebenfalls frohe Festtage,
>
>
> Jens
>
> > Am 23.12.2015 um 13:43 schrieb Michael Pippig 
> <notifications@github.com>:
> >
> > Ok, danke. Mal schauen ob ich das auch für meine anderen Repos 
> verwenden
> > kann.
> >
> > Viele Grüße und frohe Weihnachten,
> > Michael
> >
> >
> > Am 22.12.2015 um 20:21 schrieb Jens Keiner:
> > > Moin Moin,
> > >
> > > ich habe die Travis-Konfiguration aktualisiert, so dass die Tests in
> > > einem Docker-Container laufen können. Im Zweig „travis-ci"
> > > funktioniert es damit wieder.
> > >
> > > Viele Grüße,
> > >
> > >
> > > Jens
> > > > Am 21.12.2015 um 22:29 schrieb Ghislain Antony Vaillant
> > > <notifications@github.com>:
> > > >
> > > > Best might be to submit it upstream to Kitware
> > > >
> > > > —
> > > > Reply to this email directly or view it on GitHub.
> > > >
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/NFFT/nfft/issues/12#issuecomment-166708054>.
> > >
> >
> >
> > --
> > **********************************************************
> > Michael Pippig
> > eMail: michael.pippig.-tuc@gmail.com
> > WWW: http://www.tu-chemnitz.de/~potts/workgroup/pippig
> > **********************************************************
> >
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NFFT/nfft/issues/12#issuecomment-166888783>.
>


-- 
**********************************************************
Michael Pippig
eMail: 	michael.pippig.-tuc@gmail.com
WWW: 	http://www.tu-chemnitz.de/~potts/workgroup/pippig
**********************************************************

**Comment by jenskeiner** [comment]
Ok, kann ich dann machen.
